jira/client.py

```diff
@@ -17,42 +17,46 @@
 
 import collections
 import copy
+import json
+import logging
 import os
 import re
 import tempfile
-import logging
 try:  # Python 2.7+
     from logging import NullHandler
 except ImportError:
     class NullHandler(logging.Handler):
 
         def emit(self, record):
             pass
-import json
-import warnings
-import sys
-import datetime
-import time
-import xml.etree.ElementTree as etree
 import calendar
+import datetime
 import hashlib
 from numbers import Number
 import requests
+import sys
+import time
+import warnings
 
-# noinspection PyUnresolvedReferences
-from six.moves.urllib.parse import urlparse
-from six import iteritems
 from requests.utils import get_netrc_auth
+from six import iteritems
+from six.moves.urllib.parse import urlparse
 # JIRA specific resources
-from .resources import Resource, Issue, Comment, Project, Attachment, Component, Dashboard, Filter, Votes, Watchers, \
-    Worklog, IssueLink, IssueLinkType, IssueType, Priority, Version, Role, Resolution, SecurityLevel, Status, User, \
-    CustomFieldOption, RemoteLink
+from jira.resources import *  # NOQA
+import xml.etree.ElementTree as etree
```

Comment:
Please use [`defusedxml`](https://pypi.python.org/pypi/defusedxml) to access `ElementTree` instead of directly using it.